api/openapi.yaml

```
@@ -2888,6 +2905,9 @@ components:
           $ref: '#/components/schemas/ScanEstimationSummary'
         revision:
           type: integer
+        ttl:
+          description: The duration in seconds this resource should last until it is deleted.
```

What do you think about overriding the field name by adding the `x-go-name: TTL` directive?

```suggestion
          description: The duration in seconds this resource should last until it is deleted.
          x-go-name: TTL
```

---

It makes the field named in the generated code `TTL` and not `Ttl` where the later is not the way how you would name a exported field for a struct in Go.

---

Thanks for adopting.